app/com/gu/contentapi/sanity/KindleTagTest.scala

```diff
@@ -0,0 +1,32 @@
+package com.gu.contentapi.sanity
+
+import org.scalatest.concurrent.{IntegrationPatience, ScalaFutures}
+import org.scalatest.{Matchers, FlatSpec}
+
+class KindleTagTest extends FlatSpec with Matchers with ScalaFutures with IntegrationPatience {
+
+  "Tags required by the Kindle app" should "exist" in {
+    handleException{
+
+      val kindleTags = Seq(
+        "theguardian/mainsection",
+        "theguardian/mainsection/topstories",
+        "tone/advertisement-features",
+        "uk/uk",
+        "theobserver/news/worldnews")
```

Comment:
May you add those ones as well:

 * business/business
 * culture/culture
 * football/football
 * theguardian/mainsection/opinion
 * lifeandstyle/lifeandstyle
 * environment/environment